cmd/litpy/test_lit.py

```diff
@@ -17,7 +19,8 @@ def __init__(self, i):
 
         # Write a hexkey to the hexkey file
         with open(self.data_dir + "/testkey.hex", 'w+') as f:
-            f.write("1" * 64 + "\n")
+            randomString = ''.join(random.choice("abcdef"+digits) for i in range(64))
```

Comment:
It makes sense for the key to be random though; if creating multiple nodes with the same testkey.hex, they will all have the same addresses which can lead to lots of problems.